src/amp-story-player/amp-story-player-impl.js

```diff
@@ -129,6 +132,25 @@ export class AmpStoryPlayer {
       lastX: 0,
       isSwipeX: null,
     };
+
+    this.attachCallbacksToElement_();
+  }
+
+  /**
+   * Attaches callbacks to the DOM element for them to be used by publishers.
+   * @private
+   */
+  attachCallbacksToElement_() {
+    this.element_.load = this.load;
```

Comment:
What is the element? Does it have a load event?

Reply:
`<amp-story-player>` is the element, it does not have a load event until we assign it here

Reply:
What does setting the load do, then?

Reply:
This PR is creating a new `load()` method to the `<amp-story-player>` element, which internally calls `buildCallback()` and `layoutCallback()`.

Reply:
Ok. I'm imagining you're doing to do `storyPlayerEl.load()` at some point, then? That's actually going to end up calling `el.buildCallback()` and `el.layoutCallback()` (**not** `ampStoryPlayer.buildCallback()`/`layoutCallback()`). You need to bind this function to this `ampStoryPlayer` instance.

Reply:
Ohh good catch!! Done.